readme.md

```diff
@@ -243,3 +244,4 @@ To learn more, review the [privacy statement](https://go.microsoft.com/fwlink/?L
 [pnp-device-nuget]: https://www.nuget.org/packages/Microsoft.Azure.Devices.DigitalTwin.Client/
 [pnp-service-prerelease]: https://img.shields.io/nuget/vpre/Microsoft.Azure.Devices.DigitalTwin.Service.svg?style=plastic
 [pnp-service-nuget]: https://www.nuget.org/packages/Microsoft.Azure.Devices.DigitalTwin.Service/
+[pnp-device-dev-guide]: https://docs.microsoft.com/en-us/azure/iot-pnp/concepts-developer-guide-device?pivots=programming-language-csharp
```

Comment:
locales should not be included in urls. 'en-us' should be removed.

Reply:
sure thing, will update all urls in here